advanced_alchemy/utils/fixtures.py

```diff
@@ -17,11 +17,11 @@ def open_fixture(fixtures_path: Path | AsyncPath, fixture_name: str) -> Any:
     """Loads JSON file with the specified fixture name
 
     Args:
-        fixtures_path (Path): The path to look for fixtures
+        fixtures_path: :class:`pathlib.Path` or :class:`anyio.Path` The path to look for fixtures
```

Comment:
It should either omit the types or use proper syntax (`Union` or `|` ) if being explicit